modules/farm_fd2_school/src/entrypoints/FD4_Application/harvest.e2e.cy.js

```diff
@@ -12,7 +12,74 @@ describe('Tests for the Harvest form', () => {
     cy.saveSessionStorage();
   });
 
-  it('Placeholder test', () => {
-    cy.get('[data-cy="does-not-exist"]');
+  it('Check initial state of page is correct', () => {
+    //should exist
+    cy.contains('h1', 'Harvest').should('be.visible');
+
+    cy.get('[data-cy="harvest-date"]')
+      .should('be.visible')
+      .find('input[type="date"]')
+      .should('have.value', '2019-06-15');
+
+    cy.get('[data-cy="harvest-crop"]')
+      .should('be.visible')
+      .find('select')
+      .should('have.value', null);
+
+    cy.get('[data-cy="harvest-buttons"]')
+      .should('be.visible')
+      .within(() => {
```

Comment:
- It would be better to use the `data-cy` value in the `find` to get the element inside of this component.  See the documentation for the component to find the `data-cy` values that it contains.